Ev2Specs/ServiceGroupRoot/buildver.txt

```diff
@@ -0,0 +1 @@
+1.0.1
```

Comment:
Will we need to update this? When? Is it tied to the package version?

Reply:
This is the version of the Ev2 deployment. This has to be updated every time the deployment changes. In reality, since this deployment only runs the shell extension, nothing will ever change and this version is not super relevant. But ev2 still expects that.